README.md

```diff
@@ -134,10 +144,8 @@ A = [-1,   10, 0, &
 !&>
 ```
 
-
 ## Contributing / Testing
 
 The testing mechanism allows you to easily test fprettify with any Fortran project of your choice. Simply clone or copy your entire project into `fortran_tests/before` and run `python setup.py test`. The directory `fortran_tests/after` contains the test output (reformatted Fortran files). If testing fails, please submit an issue!
 
-
 [![Code Climate](https://codeclimate.com/github/pseewald/fprettify/badges/gpa.svg)](https://codeclimate.com/github/pseewald/fprettify)
```

Comment:
Should this badge go up to the top as well?

Reply:
Potentially, but given it's a D I would be happy to hide it.
There are a bunch more that are relevant for Python packages, codequality being one of them.

Reply:
@awvwgk moved to the top with the rest